src/wp-admin/media.php

```diff
@@ -9,4 +9,34 @@
  * @subpackage Administration
  */
 
-_deprecated_file( basename( __FILE__ ), '6.3.0', 'wp-admin/upload.php' );
+/** Load WordPress Administration Bootstrap */
+require_once __DIR__ . '/admin.php';
+
+$parent_file  = 'upload.php';
+$submenu_file = 'upload.php';
+
+wp_reset_vars( array( 'action' ) );
+
+switch ( $action ) {
+	case 'editattachment':
+	case 'edit':
+		// Used in the HTML title tag.
+		$title = __( 'Edit Media' );
+
+		if ( empty( $errors ) ) {
+			$errors = null;
+		}
+
+		if ( empty( $_GET['attachment_id'] ) ) {
+			wp_redirect( admin_url( 'upload.php' ) );
+			exit;
+		}
+		$att_id = (int) $_GET['attachment_id'];
+
+		wp_redirect( admin_url( "upload.php?item={$att_id}&error=throw-depreciated-media.php" ) );
```

Comment:
```suggestion
		wp_redirect( admin_url( "upload.php?item={$att_id}&error=deprecated-file-media-php" ) );
```

Small typo and a nitpick. The `.php` at the end of the URL might be a bit confusing, perhaps. Also error=deprecated seems clearer?

Reply:
Fixed in updated PR.